README.md

```diff
@@ -1,2 +1,116 @@
+  </p>
+ 
+   #### 3.Pen Lifting
+   Reck and Pinion mechanism helps us to lift the whole bot by providing perpendicular force on the wall while the pen module is in the process of rotation.
+ 
+  <p align="center">
+  <img src="Image%20and%20Videos/Mechanical%20Design%20Images/Pen%20lifting.png" alt="Pen Lifting" height="250" width="500">
+  <br><i>Pen Lifting</i>
+  </p>
+  
+## Electronics Aspect of the Design
+ In this project, OpenCV is used for image processing and the coordinates are stored in the form of a 30-30 array. Which are sent to arduino through serial transfer library. Then every coordinate is transformed into the length of two strings, which is again converted into number of steps of individual stepper motors.
+ <p align="center">
+  <img src="Image%20and%20Videos/Electronics%20Mechanism/circuit_diagram.png" alt="circuit diagram" height="500" width="810">
+  <br><i>Circuit diagram</i>
+  </p>
+ 
+## Cost Structre
+| Components | Quantity | Cost (in INR) |
+|:----------------------------------:|:-------:|:------------:|
+| Nema 11/17 Stepper Moto | 2 | 1000 |
+| Arduino Mega | 1 | 1000 |
+| MG90S Servo Motor  | 1 | 250 |
+| L293D Motor Drive Shield | 1 | 150 |
+| 28B28BYJ-48 DC 5V Stepper Motor(with ULN2003 Driver) | 1 | 150 |
+| Command Large Plastic Hooks for walls | 2 | 280 |
+| Rope - DYNAMICA XBO | 2m | 350 |
+|Total Cost|    | 3180  |
+
+## Applications
+
+* The wall art bot is planned to draw or sketch a picture which the user has in his laptop or phone.
+* This bot has three pen holders so, it can draw a picture with three different colors.
+
+## Limitations
+
+* If the two points from where the two strings are attached are not in the same level then the drawn picture will also be tilted.
+
+* The dimension of the picture should not go beyond the workspace area of the bot( although the workspace area can be increased by increasing the distance between two points where strings are attached).
+
+## Future Improvements
+Raspberry pi can be used instead of Arduino to overcome the problem with limited storage. And also an app can be developed such that user can upload the picture directly from the phone.
+#### Team members
+
+1. [Aradhya Saxena](https://github.com/aries2001)
+2. [Bhunesh Gepal](https://github.com/Bhunesh22)
+3. [Kodumuru Firdose Kouser](https://github.com/fidosekouser1902)
+4. [Shikar Gupta](https://github.com/shikhar2624)
+5. [Trishit Mondal](https://github.com/trishit-byte)
+
+
+#### Mentors
+1. [Naman Agarwalla](https://github.com/naman99-agar)
+2. [Parul Chaudhary](https://github.com/Parul253)
+
+## References
+* http://42bots.com/tutorials/28byj-48-stepper-motor-with-uln2003-driver-and-arduino-uno/
+* https://github.com/PowerBroker2/pySerialTransfer
+* https://www.youtube.com/watch?v=T0jwdrgVBBc&t=72s&ab_channel=MERTArduino%26Tech
+* https://www.youtube.com/watch?v=FbR9Xr0TVdY&ab_channel=ProgrammingKnowledge
+* https://www.hackster.io/fredrikstridsman/stringent-the-15-wall-plotter-d965ca#overview
```

Comment:
Hyperlink the References properly. No need to link the URLs again. Refer this README: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md